core/src/androidTest/java/org/hisp/dhis/android/core/data/database/AbsStoreTestCase.java

```diff
@@ -41,11 +43,15 @@
 public abstract class AbsStoreTestCase {
     private SQLiteDatabase sqLiteDatabase;
     private DatabaseAdapter databaseAdapter;
+    private String dbName = "test2.db";
 
     @Before
     public void setUp() throws IOException {
+        if(dbName!=null){
```

Comment:
@idelcano create a physical database and remove on every test convert test execution slower, really is physical database necessary? It's slower because is necessary access to the disk to remove on every test.

Reply:
@xurxodev i think it is useful and only checks if db variable is null in the normal tests. 

The database will removed only if exist, this not happen in the normal tests. 
But If you want test using a db name (for example for pull the database in the middle of the test) is necessary remove it before/after every test to be sure the database is new.
I added it because i have a lot of false failing tests in some situations after added a db name to pull it.